src/expandable-block/block.scss

```diff
@@ -167,4 +167,35 @@
       margin-left: $iui-s;
     }
   }
+
+  &.iui-borderless {
+    @include themed {
+      background-color: transparent;
+      border: initial;
+    }
+
+    .iui-header {
+      @include themed {
+        background-color: transparent;
+        border: initial;
+      }
+
+      &:hover {
+        @include themed {
+          background-color: t(iui-color-background-4);
```

Comment:
I think this hover color needs to be a bit lighter. You can inspect our borderless buttons as an example

Reply:
The screenshot posted in issue #454 showed this background color, but I'll take a look

Reply:
I think the screenshot in the issue matches my suggestion. It even explicitly says _"We would likely want to copy the hover effect that is present in our borderless buttons"_

They look similar in light theme but quite different in dark theme.
![image](https://user-images.githubusercontent.com/9084735/152600799-eeafd9a4-48b4-4980-96e5-5d3dfeb741ca.png)

```suggestion
          background-color: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-6));
```

Reply:
In light theme the hovered borderless expandable block closely matches the hovered borderless button, but in dark theme it's more obvious that they're not matching:
![Screen Shot 2022-02-04 at 3 47 13 PM](https://user-images.githubusercontent.com/849817/152600854-1d6d1766-fd3c-4575-87a0-c0152a28806e.png) ![Screen Shot 2022-02-04 at 3 47 31 PM](https://user-images.githubusercontent.com/849817/152600894-2f7b566b-c813-469e-9cc1-f1b29af2b801.png)

[Try using this background-color](https://github.com/iTwin/iTwinUI/blob/0f789cae04237e76672d28b6af665fdb961b362d/src/button/borderless.scss#L20).

Reply:
I misread matching the hover effect with the borderless buttons instead of matching with the non-borderless expandable block headers. This makes sense now

Reply:
Background color for hover has changed